typings/query-helpers.d.ts

```diff
@@ -4,12 +4,12 @@ import { Matcher, MatcherOptions } from './matches';
 
 type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
-export interface SelectorMatcherOptions extends MatcherOptions {
+export type SelectorMatcherOptions = Omit<MatcherOptions, 'selector'> & {
```

Comment:
Can you help me understand what this one does?

Reply:
The previous one tried to extend `MatcherOptions` and override `selector` property but TypeScript cannot override interface property. So to do that, you can use `Omit` to exclude `selector` and extend it again.

https://stackoverflow.com/questions/41285211/overriding-interface-property-type-defined-in-typescript-d-ts-file?answertab=votes#tab-top

Reply:
Got it, thanks 👍